web/src/admin/endpoints/connectors/agent/ConfigModal.ts

```diff
@@ -31,6 +31,9 @@ export class ConfigModal extends ModalButton {
                 .endpointsAgentsConnectorsMdmConfigCreate(this.request)
                 .then((e) => {
                     this.config = e.config;
+                })
+                .catch((exc) => {
+                    this.config = exc.toString();
                 });
```

Comment:
Something like this?

                                        .catch(async (error) => {
                                            const parsedError = await parseAPIResponseError(error);

                                            showMessage({
                                                level: MessageLevel.error,
                                                message: pluckErrorDetail(parsedError),
                                            });
                                        });